### PR TITLE
ukify: Downgrade required python version to 3.9

### PR DESCRIPTION
**Pull request by daandemeyer**

### CODE REVIEW

File: meson.build [hunk 1]
```diff
@@ -1976,13 +1976,13 @@ if run_command(python, '-c', 'import jinja2', check : false).returncode() != 0
         error('python3 jinja2 missing')
 endif
 
-python_310 = run_command(python, '-c',
-                         'import sys; sys.exit(0 if sys.version_info >= (3,10) else 1)',
-                         check : false).returncode() == 0
+python_39 = run_command(python, '-c',
+                        'import sys; sys.exit(0 if sys.version_info >= (3,9) else 1)',
+                        check : false).returncode() == 0
```
Comment (medhefgo) [inline meson.build:1981]: Why not `find_program('python3', version : '>=3.9', required : false)`?

Comment (daandemeyer) [reply meson.build:1981]: No clue @keszybz why do it this way?

**Review by behrmann — APPROVED** [review]
Haven't looked at the meson part, but the python changes LGTM.

**Comment by DaanDeMeyer** [comment]
Let's merge, we can fix the find_program() thing later

**Pull request merged**